src/lib.rs

```diff
@@ -42,17 +45,20 @@ pub fn depythonize<T>(_py: Python, _obj: PyObject) -> T {
 pub struct PythonizerError(PyErr);
 
 impl ser::Error for PythonizerError {
-    fn custom<T>(_msg: T) -> Self
+    fn custom<T>(msg: T) -> Self
     where
         T: std::fmt::Display,
     {
-        todo!()
+        let s: String = format!("{}", msg);
+        PythonizerError {
+            0: PyErr::new::<PyException, _>(s)
+        }
     }
 }
 
 impl std::fmt::Display for PythonizerError {
-    fn fmt(&self, _f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        todo!()
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{:?}", self.0)
```

Comment:
I would err on the side of using `Display` instead of `Debug` in a `Display` impl:

```suggestion
        write!(f, "{}", self.0)
```